frontend/packages/dev-console/src/components/edit-deployment/EditDeploymentPage.tsx

```diff
@@ -16,24 +17,53 @@ const EditDeploymentPage: React.FC<EditDeploymentPageProps> = ({ match, location
   const queryParams = new URLSearchParams(location.search);
   const kind = queryParams.get('kind');
   const name = queryParams.get('name');
-  const heading = t('devconsole~Edit {{kind}}', { kind });
-  const label = t('devconsole~Edit {{kind}} form', { kind });
 
-  const [deployment, loaded, loadError] = useK8sWatchResource<K8sResourceKind>({
-    kind,
-    name,
-    namespace,
-  });
+  const isNew = !name || name === '~new';
 
-  return (
+  /* Identifying the kind of the create resource (Deployment/DeploymentConfig) */
+  const createResourceType = isNew ? location.pathname.split('/')[4] : null;
```

Comment:
I think there's no need to fetch the resource type now as we are only focusing on the Deployment create form and not DeploymentConfig 
cc @divyanshiGupta what do you say ?